config/logging.php

```diff
@@ -99,6 +99,10 @@
         'emergency' => [
             'path' => storage_path('logs/laravel.log'),
         ],
+
```

Comment:
Would you please also add `bugsnag` to the `stack` driver, per the suggestion here?

https://docs.bugsnag.com/platforms/php/laravel/#laravel-5-6

Reply:
Added!